### DISCUSSION TITLE
Is sb-151 still usable?

### DISCUSSION THREAD

**Discussion by mike-uk** [GerryDazoo/Slinger#566] [Q&A] [answered]
Does the sb-151 still usable? Does it get brick like other boxes? I see it is not supported by slinger.

**Comment by GerryDazoo** [comment, marked as answer]
Make sure you've blocked outgoing port 5678. I believe it will get bricked if you're not careful. I never had one to test with but I would guess there's an excellent change it will work with Slinger.

**Reply by mike-uk** [reply]
At the moment which model can be unbricked out of all models? I think router usually block both in and out when you block a port, is that ok? It would be helpful in the readme file to state which model can be unbricked and which is not unbrickable. As I was confused when I was reading all the information as I didn't join at the beginning, I thought pro-hd need postman to be unbricked and I didn't know some model is not unbrickable and can't come back to life.
